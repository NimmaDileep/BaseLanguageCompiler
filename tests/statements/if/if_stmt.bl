var x: Int = 5;

if(x == 5) {
    print("x is 5");
} else {
    print("x is not 5");
}

if(x == 4) {
    print("x is 4");
} else {
    print("x is not 4");
}

if(x == 6) {
    print("x is 6");
} else if(x < 6) {
    print("x is less than 6");
} else {
    print("x is greater than 6");
}
