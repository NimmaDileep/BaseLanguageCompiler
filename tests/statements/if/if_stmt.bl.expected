x is 5
x is not 4
x is less than 6
